### DISCUSSION TITLE
1 Second Fountain

### DISCUSSION THREAD

**Discussion by wolfgang-ch** [mytourbook/mytourbook#1364] [General]
On Sunday, 29 September 2024, the racetrack of the Cycling WM 2024 will visit one of my favourite locations in my home area.

![image](https://github.com/mytourbook/mytourbook/assets/1283445/20096313-edcc-473b-a264-b5da4cbdf041)

On the racetrack, 400 meters before the ascent begins to the Kyburg, there is a fountain which I named a few years ago "1 second fountain" because it fills your bottle in one second

https://github.com/mytourbook/mytourbook/assets/1283445/706b9ba2-c238-4c9d-9b4f-a3b6a19bf101

Just before the ascent begins, you have to cross a wooden bridge

![DSC04387](https://github.com/mytourbook/mytourbook/assets/1283445/6d2074dc-04f4-42cd-935b-fde83146f585)

After you have claimed the hill you will have a nice view with the Kyburg in front of you

![DSC04388](https://github.com/mytourbook/mytourbook/assets/1283445/ba66c6ae-d052-4ef4-b8de-784f0c914061)

![kyburg](https://github.com/mytourbook/mytourbook/assets/1283445/a7fb39b1-2995-4963-8fbd-999ceb2de5e2)

**Comment by FJBDev** [comment]
It seems that the "1 second fountain" is faster than 1 second ! 😄
